src/it/resources/com/sun/checkstyle/test/chapter5comments/rule511blockcomments/InputCommentsIndentationSurroundingCode.java

```diff
@@ -0,0 +1,127 @@
+// comment
+package com.sun.checkstyle.test.chapter5comments.rule511blockcomments;
+
+import java.util.Arrays;
+
+// some
+public class InputCommentsIndentationSurroundingCode
+{
+    private void foo1() {
+        if (true) {
+            // here initialize some variables
+            int k = 0; // trailing comment
```

Comment:
I do not see a reason to copy all that inputs better to use few and focus on that matter for sun style description.